pages/common/adb-disconnect.md

```diff
@@ -0,0 +1,7 @@
+# adb disconnect
+
+> This command is an alias of `adb connect`.
+
+- View documentation for the original command:
```

Comment:
```suggestion
> This command has been moved.

- View documentation for `adb disconnect`:
```
Something like this?

Reply:
This would be okay in my opinion. However, I'd like to wait for other reviewers' opinions on this one.

Reply:
Can you make a suggestion for what you mean with the second line?